docs/ops.html

```diff
@@ -1789,6 +1789,16 @@ <h4 class="anchor-heading"><a id="remote_jmx" class="anchor-link"></a><a href="#
         <td>kafka.server:type=transaction-coordinator-metrics,name=partition-load-time-avg</td>
         <td>average time, in milliseconds, it took to load transaction metadata from the consumer offset partitions loaded in the last 30 seconds (including time spent waiting for the loading task to be scheduled)</td>
       </tr>
+      <tr>
+        <td>Rate of transactional verification errors</td>
+        <td>kafka.server:type=AddPartitionsToTxnManager,name=VerificationFailureRate</td>
+        <td>in steady state 0, but transient errors are expected during rolls and reassignments of the transactional state partition</td>
```

Comment:
We didn't specify what the metric actually measures. Your PR description when you added it has a nice description asL
`Rate of verifications that returned in failure either from the AddPartitionsToTxn API response or through errors in the AddPartitionsToTxnManager. In steady state this metric will report 0, but transient errors are expected during rolls and reassignments of the transactional state partition`